Ecosystem Partners/Marketplace Guides/getting-started-with-avni-software-defined-cloud-partner-template.md

```diff
@@ -0,0 +1,158 @@
+
+For the first time in the cloud industry, Avni has made it possible to automate and simplify every aspect of infrastructure design, build-out, re-architect, and scale private, hybrid, and public clouds. With the Avni SDC software, you can now deploy apps in minutes – not in weeks or months – and see immediate, measurable and significant savings in OpEx and CapEx.
+
+Customers can use Avni for Dev/Test, DevOps automation, existing business-critical multi-tier apps in production, scale-out micro-services based apps, etc. In all cases, existing 
+
+The Avni SDC software is the first solution to integrate all the critical technologies so that you can deploy your existing apps in minutes in any cloud. For more information on SDC, use the contact information above.
+
+### Audience
+- CenturyLink Cloud Users
+
+### Impact
+After reading this article, the user should feel comfortable deploying the Avni SDC technology on CenturyLink Cloud.  Avni has provided a Virtual Appliance - what we call a Partner Template - that can be deployed to your CenturyLink Cloud account via a Service Task.
+
+This deployment process for Partner Templates currently requires manual interaction via the Service Task process, but will be further automated in future releases of the CenturyLink Cloud Platform.
+
+If you are interested in seeing this type of Partner Template deployment as an automated feature in the future, please share your input with us at [features@centurylinkcloud.com](mailto:features@centurylinkcloud.com)
+
+### Prerequisite
+- User requires access to the SDC Cloud platform as an authorized user. 
+- Access to the CenturyLink Cloud platform as an authorized user.
+- Identify a Network VLAN you want the Avni SDC VM to reside on
+
+### Postrequisite
+- If you want to access your Avni SDC over the internet, please perform the following tasks once your Avni SDC has been deployed to your account:
+
+1. [Add a Public IP](../../Network/how-to-add-public-ip-to-virtual-machine.md) to your server through Control Portal
+
+2. If required, [allow incoming traffic for the admin port](../../Network/how-to-add-public-ip-to-virtual-machine.md) by clicking on the Servers Public IP in Control Portal.  Warning: Please make sure your firewall rules are properly configured to only allow specific source traffic.  If you do not configure source traffic rules you risk exposing your VM to the entire internet.  Note: When accessing your VM for the first time or for any administration, we recommend you connect to your CenturyLink Cloud environment via Client VPN.
+
+3. Allow incoming traffic for the following ports
+- TCP Ports: 80, 443, 8090
+
+![Open Firewall Port](https://t3n.zendesk.com/attachments/token/1Ufw0JjIWW8XfASYLh4x3Irl9/?name=port.jpg)
```

Comment:
Please don't link to images on our zendesk if you need the image make a copy in the repository